### ISSUE TITLE
EigenPy seems to return wrong values for RowMajor matrices with 1 row

### ISSUE THREAD

**Issue by cmastalli** [stack-of-tasks/eigenpy#265]
This error can be reproduced in Crocoddyl with the double-pendulum example: https://github.com/loco-3d/crocoddyl/blob/master/examples/double_pendulum.py

When I return the feedback gains (using `solver.K`), I observe that the first value reported by the bindings is the last column of the K matrix, e.g
```python
# Here code of the Crocoddyl example

# The code below returns array([-9.06294536e-02,  3.71715372e-05,  7.70249597e-05,  4.56026493e-05]),
# but the real value is [ 1.51564e-01 -7.09909e-01 -4.30023e-02 -9.06295e-02]
# (value obtained using std::cout in the c++ code)
print(solver.k[0])
```

**Comment by cmastalli** [comment]
**Update**:
The above issue doesn't occur when we switch to NumPy matrices.

**Comment by cmastalli** [comment]
@jcarpent -- I have prepared a minimal example that depends on eigenpy only.
In the zip file, you have a Makefile that needs so you can compile the code with your setup.
Please read the "readme" file for more instructions.

[eigenpy_bug.zip](https://github.com/stack-of-tasks/eigenpy/files/7982800/eigenpy_bug.zip)

**Comment by jcarpent** [comment]
Solved by #266
